src/main/java/me/rayzr522/jsonmessage/JSONMessage.java

```diff
@@ -792,7 +859,11 @@ static void set(String field, Object obj, Object value) {
                 e.printStackTrace();
             }
         }
-
+        
+        public static String getStringVersion() {
```

Comment:
```suggestion
        public static String getVersionString() {
```

Reply:
This is from the changes I mentioned in my comment.
I don't even see any usage here, as this was used for checks like `if(ReflectionHelper.getStringVersion().equals("1_16_R1"))` or something like that which made it fairly restricted on versions before or after 1.16.
I'll remove this tbh.